app/javascript/components/game/Game.js

```diff
@@ -1,97 +1,98 @@
-import React, { Component } from 'react'
-import GameForm from '../gameForm'
-import Cell from '../Cell'
-import ResumeGame from '../ResumeGame'
-import '../game/game.css'
+import React, { Component } from "react";
+import GameForm from "../gameForm";
+import Cell from "../Cell";
+import ResumeGame from "../ResumeGame";
+import "../game/game.css";
 
-class Game extends Component{
-    state = {
-        messages: {
-          board: '',
-          win:'',
-          draw:''
-        },
-        id: ''
-      }
+class Game extends Component {
+  state = {
+    messages: {
+      board: "",
+      win: "",
+      draw: "",
+    },
+    id: "",
+  };
 
-      addGame = (form) => { 
-        const initial_game =  {
-          board: '',
-          win:'',
-          draw:''
-        }      
-        this.setState({
-            id: form.id,
-            messages: initial_game
-          });
-          localStorage.setItem('gameId', form.id)
+  addGame = (form) => {
+    const initial_game = {
+      board: "",
+      win: "",
+      draw: "",
+    };
+    this.setState({
+      id: form.id,
+      messages: initial_game,
+    });
+    localStorage.setItem("gameId", form.id);
```

Comment:
Could we inject `localStorage` in this component as well to make it easier to test?